packages/dynamite/dynamite_end_to_end_test/lib/some_of.openapi.dart

```diff
@@ -29,6 +33,79 @@ class Client extends DynamiteClient {
           cookieJar: client.cookieJar,
           authentications: client.authentications,
         );
+
+  /// Returns a [Future] containing a [DynamiteResponse] with the status code, deserialized body and headers.
+  /// Throws a [DynamiteApiException] if the API call does not return an expected status code.
+  ///
+  /// Parameters:
+  ///   * [oneOf]
+  ///   * [anyOf]
+  ///
+  /// Status codes:
+  ///   * default
+  ///
+  /// See:
+  ///  * [$getRaw] for an experimental operation that returns a [DynamiteRawResponse] that can be serialized.
+  Future<DynamiteResponse<void, void>> $get({
+    final ({bool? $bool, String? string})? oneOf,
+    final ({bool? $bool, String? string})? anyOf,
+  }) async {
+    final rawResponse = $getRaw(
+      oneOf: oneOf,
+      anyOf: anyOf,
+    );
+
+    return rawResponse.future;
+  }
+
+  /// This method and the response it returns is experimental. The API might change without a major version bump.
+  ///
+  /// Returns a [Future] containing a [DynamiteRawResponse] with the raw [HttpClientResponse] and serialization helpers.
+  /// Throws a [DynamiteApiException] if the API call does not return an expected status code.
+  ///
+  /// Parameters:
+  ///   * [oneOf]
+  ///   * [anyOf]
+  ///
+  /// Status codes:
+  ///   * default
+  ///
+  /// See:
+  ///  * [$get] for an operation that returns a [DynamiteResponse] with a stable API.
+  @experimental
+  DynamiteRawResponse<void, void> $getRaw({
+    final ({bool? $bool, String? string})? oneOf,
+    final ({bool? $bool, String? string})? anyOf,
+  }) {
+    final pathParameters = <String, dynamic>{};
+    final queryParameters = <String, dynamic>{};
+    final headers = <String, String>{};
+    Uint8List? body;
+
+    if (oneOf != null) {
+      queryParameters['oneOf'] = oneOf.toString();
+    }
+    if (anyOf != null) {
+      queryParameters['anyOf'] = anyOf.toString();
+    }
```

Comment:
I assume this should call serialize instead, but I'm not sure if that would be enough? @Leptopoda could you have a look at this and fix it (in a separate PR)?

Reply:
Do you prefer merging this already (with the broken serialization) or fixing it in this PR as well?

Reply:
If we need it for 28 I'll review it and we can merge.

If not I'd rather add unit tests with a mocked HttpClient first.
I'll open an issue for that.

Reply:
I'll do both. Should be straightforward.